zerver/webhooks/ansibletower/view.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, Iterable, Optional, List
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile
+
+import ujson
+import operator
+
+ANSIBLETOWER_SUBJECT_TEMPLATE = "{name}"
+
+ANSIBLETOWER_DEFAULT_MESSAGE_TEMPLATE = ("Job Type: {friendly_name}\n"
+                                         "ID: [{id}]({url})\n"
+                                         "Name: {name}\n"
+                                         "Status: {status}")
+
+ANSIBLETOWER_JOB_MESSAGE_TEMPLATE = ("Job Type: {friendly_name}\n"
+                                     "ID: [{id}]({url})\n"
+                                     "Name: {name}\n"
+                                     "Status: {status}\n"
+                                     "Hosts:\n"
+                                     "{hosts_final_data}")
+
```

Comment:
As for messages, we want to aim for a conversational tone in our notifications. Just listing out the properties of a job doesn't read too well.

So, I would try and experiment with templates like the following:
```
Job [#{id} {name}]({url}) was {status}.
Hosts: {hosts_final_data}.
```

This is more like what we already do with some of our webhooks! :)

Reply:
Yeah. I've went with something similar to this, looks a lot cleaner to read now. Updated the screenshot in the docs as well to match the new layout!